internal/kubeclient/config.go

```diff
@@ -0,0 +1,20 @@
+package kubeclient
+
+import (
+	"os"
+
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var overrides = clientcmd.ConfigOverrides{}
+
+var (
+	explicitPath string
+)
+
+func NewClientConfig() clientcmd.ClientConfig {
+	rules := clientcmd.NewDefaultClientConfigLoadingRules()
+	rules.DefaultClientConfig = &clientcmd.DefaultClientConfig
+	rules.ExplicitPath = explicitPath
```

Comment:
I can't see where the package-level `explicitPath` variable ever takes on a new value, so isn't this always equivalent to `rules.ExplicitPath = ""`? (And isn't empty string the default value of `rules.ExplicitPath` anyway?

Reply:
I added `explicitPath` for letting CLI to override it, but I think we can add it in the later PR. Will remove in this PR.